src/services/dashboards/dashboard-detail/store/dashboard-detail-info.ts

```diff
@@ -166,6 +167,11 @@ export const useDashboardDetailInfoStore = defineStore('dashboard-detail-info',
     };
 
     const setDashboardInfo = (dashboardInfo: DashboardModel) => {
+        dashboardInfo.variables_schema = {
+            properties: { ...managedDashboardVariablesSchema.properties, ...dashboardInfo.variables_schema.properties },
+            order: union(managedDashboardVariablesSchema.order, dashboardInfo.variables_schema.order),
+        };
```

Comment:
suggestion: I think this should be changed to

```suggestion
        state.variablesSchema = {
            properties: { ...managedDashboardVariablesSchema.properties, ...dashboardInfo.variables_schema.properties },
            order: union(managedDashboardVariablesSchema.order, dashboardInfo.variables_schema.order),
        };
```